subprojects/docs/src/docs/userguide/running-builds/configuration_cache.adoc

```diff
@@ -204,6 +204,43 @@ If you need to invalidate the cache, simply delete that directory:
 Configuration cache entries are checked periodically (at most every 24 hours) for whether they are still in use.
 They are deleted if they haven't been used for 7 days.
 
+[[config_cache:usage:encryption]]
+=== Encrypting the cache
+
+The configuration cache stores any data set up during the configuration phase.
+As such, if your build configuration includes any sort of sensitive information, you should ensure
+your configuration cache is encrypted.
+
+For Gradle to be able to encrypt the data in the cache, you must provide Gradle with an AES 128-bit key
+via the `GRADLE_ENCRYPTION_KEY` environment variable (see <<#sec:gradle_environment_variables, see Environment variables>>).
+
+==== Generating an encryption key that is compatible with GRADLE_ENCRYPTION_KEY
+
+There are many different ways of producing an encryption key that is compatible with Gradle.
+One way is by using a command like this, which should work on Linux and MacOSx, or even on Windows, if using a tool like Cygwin:
+
+```
+echo $(openssl enc -aes-128-cbc -k secret -P | grep key= | sed s/key=// | base64)
+```
+
+You can then use the key produced by that command (which is encoded as a Base64 string) and set it as the value of the
+`GRADLE_ENCRYPTION_KEY` environment variable however you prefer configuring environment variables in your OS.
```

Comment:
🧐 
```suggestion
`GRADLE_ENCRYPTION_KEY` environment variable; however, you prefer configuring environment variables in your OS.
```

Reply:
"however" as in "whichever way" - not usual English I gather?